### ISSUE TITLE
question about batch size 

### ISSUE THREAD

**Issue by seungjunlee96** [rosinality/stylegan2-pytorch#124]
From the discriminator implementation (model.py), the default value for `self.stddev_group` is 4.

This gives error in reshaping(or view) operation if the batch size not set multiple of 4 ( and I used multiple GPUs)

below is original implementation of yours 
```python
    def forward(self, input, is_feature = False):
        out = self.convs(input)

        batch, channel, height, width = out.shape
        group = min(batch,self.stddev_group)
        stddev = out.view(
            group, -1, self.stddev_feat, channel // self.stddev_feat, height, width
        )
```

but I found that when I changed the code 
` group = min(batch,self.stddev_group)`
into 
`group = max(batch,self.stddev_group)`
it does not give error.

I wonder this would be problematic?

**Comment by rosinality** [comment]
I guess it will be not very problematic, but I don't know exactly how it will affect the results.

**Comment by seungjunlee96** [comment]
Thanks :) 
I could not find a difference in the generated images (at least in my visual inspection)
